staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -190,7 +191,7 @@ func (a *Admission) ValidateConfiguration() error {
 
 // Validate admits an API request.
 // The objects in admission attributes are expected to be external v1 objects that we care about.
-func (a *Admission) Validate(ctx context.Context, attrs Attributes) *admissionv1.AdmissionResponse {
+func (a *Admission) Validate(ctx context.Context, attrs api.Attributes) *admissionv1.AdmissionResponse {
```

Comment:
this doesn't look right, these Attributes refer to https://github.com/kubernetes/kubernetes/blob/cf6ba6096f94017eda4644e1bebe169dc147ef78/staging/src/k8s.io/pod-security-admission/admission/attributes.go#L26

Reply:
oh I see you moved the interface to a dedicated package.